qark/plugins/logging_plugin.py

```diff
@@ -1,17 +1,29 @@
-import sys, os, re
-
-sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
-
 from yapsy.IPlugin import IPlugin
 from plugins import PluginUtil
 from modules import common
 from lib.pubsub import pub
+import sys
+import os
+import re
 import lib.plyj.parser as plyj
 import lib.plyj.model as m
 
+sys.path.insert(0, os.path.dirname(os.path.realpath(__file__)) + '../lib')
 
-class LoggingIssuesPlugin(IPlugin):
+string = ("{} logs are detected\nThis may allow potential leakage of information from Android applications.\n"
```

Comment:
Might want to make this upper case - easier to tell what's a constant and what isn't